src/components/GenuineCheck.js

```diff
@@ -109,7 +110,7 @@ class GenuineCheck extends PureComponent<Props, State> {
     // Preload things in parallel
     Promise.all([
       // Step dashboard, we preload the applist before entering manager while we're still doing the genuine check
-      manager.getAppsList(deviceInfo),
+      manager.getAppsList(deviceInfo, getEnv('MANAGER_DEV_MODE')),
```

Comment:
was this accidentally added? i don't think there is such param and if there is, it's not scope of what this PR fixes?